### PR TITLE
Update to last version of liquidjs (wasm)

### LINKED ISSUE #26: Support latest liquidjs-lib with async blindOutputs and blindOutputsByIndex

### PR DESCRIPTION
**Pull request by louisinger**
This PR updates LDK in order to use the latest changes of liquidjs.

- update dependencies.
- `tryToUnblindOutput` becomes `unblindOutput` (because the method does not "try").
- RIP the last `unblindOutput` in utils.ts --> replaced by the new liquidjs function: `confidential.UnblindOutputWithKey`.
- update tests.

@tiero @altafan please review

it closes #26

### CODE REVIEW

File: src/explorer/esplora.ts [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
+  UnblindedOutputInterface,
+  UtxoInterface,
   isBlindedOutputInterface,
 } from '../types';
-import axios from 'axios';
-import { EsploraTx } from './types';
+import { Transaction, TxOutput, confidential } from 'liquidjs-lib';
 import { isConfidentialOutput, toAssetHash, toNumber } from '../utils';
 
+import { EsploraTx } from './types';
```
Comment (altafan) [inline src/explorer/esplora.ts:14]: Remove empty line

File: src/explorer/esplora.ts [hunk 2]
```diff
@@ -249,19 +250,23 @@ async function unblindTransactionPrevoutsAndOutputs(
   return tx;
 }
 
-export function tryToUnblindOutput(
+export async function unblindOutput(
```
Comment (altafan) [inline src/explorer/esplora.ts:253]: so this now throws if failing to unblind, right?

**Review by altafan — CHANGES_REQUESTED** [review]
Seem good to me... Can you remove all those blank lines between imports tho? There are many in several files

File: src/explorer/esplora.ts (hunk 2 above)
Comment (louisinger) [reply src/explorer/esplora.ts:253]: yes (it always throw an error if failing to unblind)

**Pull request merged**